conf/routes

```diff
@@ -15,15 +15,16 @@ POST        /admin/users/edit                                  @femr.ui.controll
 GET         /admin/users/edit/:id                              @femr.ui.controllers.admin.UsersController.editGet(id: Integer)
 POST        /admin/users/toggle/:id                            @femr.ui.controllers.admin.UsersController.toggleUser(id: Integer)
 GET         /admin/users                                       @femr.ui.controllers.admin.UsersController.manageGet()
-GET         /admin/inventory                                   @femr.ui.controllers.admin.InventoryController.manageGet()
-GET         /admin/inventory/custom                            @femr.ui.controllers.admin.InventoryController.customGet()
-POST        /admin/inventory/custom                            @femr.ui.controllers.admin.InventoryController.customPost()
-GET         /admin/inventory/existing                          @femr.ui.controllers.admin.InventoryController.existingGet()
-POST        /admin/inventory/existing                          @femr.ui.controllers.admin.InventoryController.existingPost()
+POST        /admin/inventory                                   @femr.ui.controllers.admin.InventoryController.managePost()
+GET         /admin/inventory/custom/:tripId                    @femr.ui.controllers.admin.InventoryController.customGet(tripId: Integer)
+POST        /admin/inventory/custom/:tripId                    @femr.ui.controllers.admin.InventoryController.customPost(tripId: Integer)
+GET         /admin/inventory/existing/:tripId                  @femr.ui.controllers.admin.InventoryController.existingGet(tripId: Integer)
+POST        /admin/inventory/existing/:tripId                  @femr.ui.controllers.admin.InventoryController.existingPost(tripId: Integer)
 GET         /admin/inventory/export/:tripId                    @femr.ui.controllers.admin.InventoryController.exportGet(tripId: Integer)
 POST        /admin/inventory/delete/:id/:tripId                @femr.ui.controllers.admin.InventoryController.ajaxDelete(id: Integer, tripId: Integer)
 POST        /admin/inventory/editCurrent/:id/:tripId           @femr.ui.controllers.admin.InventoryController.ajaxEditCurrent(id: Integer, tripId: Integer)
 POST        /admin/inventory/editTotal/:id/:tripId             @femr.ui.controllers.admin.InventoryController.ajaxEditTotal(id: Integer, tripId: Integer)
+GET         /admin/inventory/:id                               @femr.ui.controllers.admin.InventoryController.manageGet(id: Integer)
```

Comment:
can this go above line 18 or are you running into conflicts with the routing?

Reply:
I was having issues with this at one point - I think because it was getting a string from somewhere instead of an integer. I moved it back to line 18 and everything still seems to be working fine now.